boards/nxp/mimxrt595_evk/doc/index.rst

```diff
@@ -185,7 +185,7 @@ To build the hello_world sample for the RT500 DSP core:
 
 .. code-block:: shell
 
-   $ west build -b nxp_adsp_rt595 samples/hello_world
+   $ west build -b mimxrt595_evk//f1 samples/hello_world
```

Comment:
Use full board names and identifiers in all examples `mimxrt595_evk/rt595/f1`

Reply:
Changed- I understand being verbose here, but where will we document the "short name" feature of using `mimxrt595_evk//f1`? I find this pretty useful, and want to make sure customers are aware of it.

Reply:
It will be in the hwmv2 documentation